### ISSUE TITLE
Learn How to Talk About a Typical Workday and Tasks: Step 108 - Required word is never uttered

### ISSUE THREAD

**Issue by lasjorg** [freeCodeCamp/freeCodeCamp#54172]
### Describe the Issue

The word "ethical" is never spoken, only "ethics".

> Linda learns that compliance helps maintain safety and ethical standards.

"Ethical standards" is an umbrella term. It would be difficult for a language learner to reach that conclusion without the word ever spoken in context.

---

code: https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/657e61ba2a53680d1a540bfe.md

### Affected Page

https://www.freecodecamp.org/learn/a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/task-108

### Your code

```



```

### Expected behavior

The exact required word to be spoken and used in context.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

Forum: https://forum.freecodecamp.org/t/learn-how-to-talk-about-a-typical-workday-and-tasks-task-108/681292

**Comment by lasjorg** [comment]
I updated the issue. I think we might be requiring the wrong word here.

**Comment by Sembauke** [comment]
Yes, you are correct. It might make sense to advanced English speakers but no so much for beginners. 

Let change:
>Linda learns that compliance helps maintain safety and _ standards

To 
> Linda learns that compliance helps maintain security and safety standards.

Meaning Safety becomes the blank word so security is not appearing twice as a blank.

**Comment by lasjorg** [comment]
It makes more sense to me with the word `security`, as that is how it is used in the spoken sentence.

> Great question! We have to comply with data protection regulations, security standards, and our company's code of conduct.

The only time "standards" is spoken is with "security" before it, so when you reach that part of the fill in the blanks

> Linda learns that compliance helps maintain safety and _ standards

I think it is only logical to assume that the word would be "security"

> Linda learns that compliance helps maintain safety and security standards
